Yoast/YoastSEO.js
# Issue1219 typescriptify app.js

## Summary

This PR can be summarized in the following changelog entry:
Added support for TypeScript.

## Relevant technical choices:

* TypeScript files have to be created inside the 'src' folder.
* The javascript file in the 'build' folder will be overwritten when a JavaScript file and a TypeScript file have the same name.

## Test instructions

This PR can be tested by following these steps:

* Create a TypeScript file in the src folder and execute the grunt build command from the command line. All JavaScript code and TypeScript code should be merged into the 'build' folder.

Fixes #1219
